client/account-recovery/account-recovery-root/index.jsx

```diff
@@ -49,6 +51,10 @@ const getPageInfo = ( translate, step ) => {
 			trackerTitle: 'Account Recovery > Reset Password Sms',
 			documentHeadTitle: concatHeadTitle( translate( 'Reset Password' ), translate( 'SMS Message' ) ),
 		},
+		[ STEPS.RESET_PASSWORD_CONFIRM ]: {
+			trackerTitle: 'Account Recovery > New Password',
+			documentHeadTitle: translate( 'Reset Password < New Password' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **79** times:
`translate( 'Repeat New Password' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).